index.ts

```diff
@@ -1,32 +1,39 @@
-import { device, DeviceOptions } from 'aws-iot-device-sdk';
-import { Subject, Observer, Observable } from 'rxjs';
-import * as AWS from 'aws-sdk';
-import { v4 as uuidV4 } from 'uuid';
-import * as pako from 'pako';
+import { device, DeviceOptions } from "aws-iot-device-sdk";
+import { Subject, Observer, Observable } from "rxjs";
+import * as AWS from "aws-sdk";
+import { v4 as uuidV4 } from "uuid";
+import * as pako from "pako";
 
 export default class AwsIot {
-  public readonly events = new Subject<IotEvent>();
+  get events(): Observable<IotEvent> {
+    return this._events.asObservable();
+  }
+
+  get topicsForTest(): Array<IDeferredTopic> {
```

Comment:
Borde inte vara nödvändigt, i testet kan du istället köra `awsIot['_deferredTopics']`, det kringgår access modifiers men type safety behålls

Reply:
Yeah good point, not sure why I didn't think about that, just got annoyed by TS at that point. Will fix.